cli/src/deno.rs

```diff
@@ -208,9 +208,12 @@ impl ModuleLoader for ExtensionsModuleLoader {
             };
 
             if should_transpile {
-                let transpiled =
-                    source_mapper.transpile(module_specifier.to_string(), code, media_type)?;
-                code = transpiled.text.clone();
+                let transpiled = source_mapper.transpile(
+                    module_specifier.to_string(),
+                    code.clone(),
+                    media_type,
+                )?;
+                code.clone_from(&transpiled.text);
```

Comment:
What was the clippy warning here? Seems odd to me to clone the code just to overwrite it immediately.

Reply:
This was the error, I updated the fix.

```bash
error: assigning the result of `Clone::clone()` may be inefficient
   --> cli/src/deno.rs:213:17
    |
213 |                 code = transpiled.text.clone();
    |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: use `clone_from()`: `code.clone_from(&transpiled.text)`
    |
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#assigning_clones
    = note: `-D clippy::assigning-clones` implied by `-D warnings`
    = help: to override `-D warnings` add `#[allow(clippy::assigning_clones)]`

error: could not compile `phylum-cli` (lib) due to 1 previous error
warning: build failed, waiting for other jobs to finish...
error: could not compile `phylum-cli` (lib test) due to 1 previous error
```